src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days < SHORT_TERM) {
+    return basePrice;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return basePrice - SMALL_DISCOUNT;
```

Comment:
The condition `days >= SHORT_TERM` is redundant because if the execution reaches this point, it means that the previous condition `days < SHORT_TERM` was false, which implies `days >= SHORT_TERM`. You can simplify the condition to just `days < LONG_TERM`.